sklearn/ensemble/tests/test_bagging.py

```diff
@@ -663,3 +663,28 @@ def test_oob_score_removed_on_warm_start():
     clf.fit(X, y)
 
     assert_raises(AttributeError, getattr, clf, "oob_score_")
+
+
+def test_consistent_index_sampling():
+    # Make sure randomly drawn indices generated at fit time are identically 
+    # reproduced at a later time (for use in, e.g., OOB scoring)
+    X, y = make_hastie_10_2(n_samples=20, random_state=1)
+    
+    bagging = BaggingClassifier(KNeighborsClassifier(), max_samples=0.5, max_features=0.5)
+    bagging.fit(X, y)
+
+    feat_inds_list = bagging.estimators_features_
+    feat_samp_inds_gen = bagging._get_estimators_data_draws()
```

Comment:
When I call 
```python
feat_inds_list = bagging.estimators_features_
```
I'm retrieving the exact list of feature indices that were created within `_parallel_build_estimators()` during the call to `BaseBagging._fit()`.

When I call
```python
feat_samp_inds_gen = bagging._get_estimators_data_draws()
```
I'm dynamically re-generating both the sample and feature indices. I could also call `BaseBagging.estimators_samples_` as a way of dynamically generating just the sample indices, however there is no static version to compare them to.

The only statically generated set of indices left at this point is `BaseBagging.estimators_features_`. Thus I compare the values stored in that attribute against its dynamically generated counterpart, which can only be accessed by calling `BaseBagging._get_estimators_data_draws()`.